### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by amashutikov**
- [DEMO LINK](https://amashutikov.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://amashutikov.github.io/layout_product-cards/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,13 +1,56 @@
...
+      src="/images/imac.jpeg"
+      alt="apple a1419 imac image"
+      class="card__image"
+      >
+
+      <h2 class="card__name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+
+      <p class="card__code">Product code: 195434</p>
+
+      <div class="card__review">
```
Comment (growingananas) [inline src/index.html:28]: Suggestion: better to use 'ul-ui'  instead of 'div' to maintain semantics

File: src/styles/card_style.css [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image {
+  display: flex;
+  width: 160px;
+  height: 134px;
+  margin-top: 32px;
+  margin-left: 19px;
```
Comment (growingananas) [inline src/styles/card_style.css:13]: instead of using 'margin-left' you could use display:  flex, align-self: center

Comment (amashutikov) [reply src/styles/card_style.css:13]: Ok, thanks for advice)

File: src/index.html [hunk 3]
```diff
@@ -1,13 +1,56 @@
...
+      href="https://fonts.gstatic.com" crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <article class="card" data-qa="card">
+      <img
+      src="/images/imac.jpeg"
+      alt="apple a1419 imac image"
+      class="card__image"
```
Comment (silchyn) [inline src/index.html:21]:
```suggestion
        src="/images/imac.jpeg"
        alt="apple a1419 imac image"
        class="card__image"
```

File: src/index.html [hunk 4]
```diff
@@ -1,13 +1,56 @@
...
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+        </ul>
+
+        <p class="card__review-number">Reviews: 5</p>
+      </div>
+
+      <div class="card__price">
+        <p class="card__text">Price:</p>
+        <p class="card__item">$2,199</p>
```
Comment (silchyn) [inline src/index.html:42]: item?

File: src/index.html [hunk 5]
```diff
@@ -1,13 +1,56 @@
...
+      </div>
+
+      <div class="card__button">
+        <a
+          href="#buy"
+          class="card__link"
+          data-qa="hover"
+          >
```
Comment (silchyn) [inline src/index.html:50]:
```suggestion
        <a
          href="#buy"
          class="card__link"
          data-qa="hover"
        >
```

File: src/styles/card_style.css [hunk 6]
```diff
@@ -0,0 +1,93 @@
+.card {
+  font-family: "Roboto", sans-serif;
+  width: 198px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image {
+  display: flex;
```
Comment (silchyn) [inline src/styles/card_style.css:9]: do you need flex here?

File: src/styles/card_style.css [hunk 7]
```diff
@@ -0,0 +1,93 @@
...
+  width: 160px;
+  height: 134px;
```
Comment (silchyn) [inline src/styles/card_style.css:11]: so we stretch image?

File: src/styles/card_style.css (hunk 2 above)
Comment (silchyn) [inline src/styles/card_style.css:13]: and what if card's width will increase?

File: src/styles/card_style.css [hunk 8]
```diff
@@ -0,0 +1,93 @@
...
+  box-sizing: border-box;
+  background-color: #00acdc;
+  display: inline-block;
+  width: 100%;
+  height: 100%;
+  text-decoration: none;
+  color: #fff;
+  font-weight: 700;
+  font-size: 14px;
+  border: 1px solid #00acdc;
```
Comment (silchyn) [inline src/styles/card_style.css:51]:
```suggestion
  border: 1px solid transparent;
```

File: src/styles/stars.css [hunk 9]
```diff
@@ -0,0 +1,27 @@
+.stars {
+  margin: 0;
+  padding: 0;
```
Comment (silchyn) [inline src/styles/stars.css:3]: maybe this should go for `ul`?

File: src/index.html [hunk 10]
```diff
@@ -1,13 +1,58 @@
...
+  </head>
+  <body>
+    <article class="card" data-qa="card">
+      <div class="card__image">
+        <img
+          src="/images/imac.jpeg"
+          alt="apple a1419 imac image"
+          class="card__image-item"
+        >
+      </div>
```
Comment (silchyn) [inline src/index.html:24]:
```suggestion
      <div class="card__image-container">
        <img
          src="/images/imac.jpeg"
          alt="apple a1419 imac image"
          class="card__image"
        >
      </div>
```

File: src/index.html [hunk 11]
```diff
@@ -1,13 +1,58 @@
...
+
+      <h2 class="card__name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
```
Comment (silchyn) [inline src/index.html:26]:
```suggestion
      <h2 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
```

**Review by silchyn — APPROVED** [review]

**Pull request closed without merging**